apps/web/components/ui/AuthContainer.tsx

```diff
@@ -17,27 +17,25 @@ interface Props {
 
 export default function AuthContainer(props: React.PropsWithChildren<Props>) {
   return (
-    <div className="flex min-h-screen flex-col justify-center border-gray-200 bg-neutral-50 py-12 sm:px-6 lg:px-8">
```

Comment:
This diff is here since the v1 and v2 file had the same name, I moved the v2 to this place and deleted v1. So here you see the differences between these two 🙂